Assignment6.Rmd

```diff
@@ -23,18 +23,19 @@ library(rpart)
 #Upload the data sets MOOC1.csv and MOOC2.csv
 M1 <- read.csv("MOOC1.csv", header = TRUE)
 
-M2 <- 
+M2 <- read.csv("MOOC2.csv", header = TRUE)
 
 ```
 
 #Decision tree
 ```{r}
 #Using the rpart package generate a classification tree predicting certified from the other variables in the M1 data frame. Which variables should you use?
 
-c.tree1 <- 
+c.tree1 <- rpart(certified ~ forum.posts + grade + assignment, method ="class", data=M1, control = rpart.control(minsplit = 1, minbucket = 1, cp=0.001))
```

Comment:
You may need to `scale()` your variables as `forum.posts` has many more increments and will be preferenced by the algorithm.